common/src/main/scala/com/gu/multimedia/storagetier/vidispine/VidispineCommunicator.scala

```diff
@@ -20,7 +20,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class VidispineCommunicator(config:VidispineConfig) (implicit ec:ExecutionContext, mat:Materializer, actorSystem:ActorSystem){
   private final val logger = LoggerFactory.getLogger(getClass)
 
-  private final val maxFilesToFetch = 10000
+  private final val maxFilesToFetch = 20000
```

Comment:
Can you add a comment saying this has to correlate with the opensearch value that was changed

Reply:
Yes. Done.